src/sinks/new_relic/config.rs

```diff
@@ -210,9 +210,7 @@ impl NewRelicCredentials {
             },
             NewRelicApi::Metrics => match self.region {
                 NewRelicRegion::Us => Uri::from_static("https://metric-api.newrelic.com/metric/v1"),
-                NewRelicRegion::Eu => {
-                    Uri::from_static("https://metric-api.eu.newrelic.com/metric/v1")
-                }
+                NewRelicRegion::Eu => Uri::from_static("https://metric-api.eu.newrelic.com/metric/v1"),
```

Comment:
While this looks cleaner, it doesn't survive `cargo fmt`, which we require on all code.

Reply:
My bad, it wasn't necessary at all, I just made it instinctively because is my coding style. I'll change it.